src/fs/storage.ts

```diff
@@ -93,11 +94,15 @@ export class SessionStorageFSStorage implements FSStorage {
   }
 
   async setProjectName(projectName: string) {
-    this.storage.setItem("projectName", projectName);
+    if (projectName === undefined) {
+      this.storage.removeItem(projectNameKey);
+    } else {
+      this.storage.setItem(projectNameKey, projectName);
+    }
```

Comment:
Oops, I'd changed the signature to support undefined at some point, reverted it, and left this nonsense in.

```suggestion
    this.storage.setItem(projectNameKey, projectName);
```